android/app/build.gradle

```diff
@@ -46,11 +45,56 @@ android {
         versionName = flutterVersionName
     }
 
-    buildTypes {
+    def keystoreProperties = new Properties()
+    def keystorePropertiesFile = rootProject.file("key.properties")
+    if (keystorePropertiesFile.exists()) {
+        keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
+    }
+
+    signingConfigs {
+        debug {
+            String filePath = keystoreProperties["storeFileDebug"]
+            if (filePath != null) {
+                keyAlias = keystoreProperties["keyAliasDebug"]
+                keyPassword = keystoreProperties["keyPasswordDebug"]
+                storeFile = file(filePath)
+                storePassword = keystoreProperties["storePasswordDebug"]
+            } else {
+                println("No storeFile provided, debug builds are using your local debug keystore")
+            }
+        }
+
         release {
-            // TODO: Add your own signing config for the release build.
-            // Signing with the debug keys for now, so `flutter run --release` works.
+            String filePath = keystoreProperties["storeFile"]
+            if (filePath != null) {
+                keyAlias = keystoreProperties["alias"]
+                keyPassword = keystoreProperties["password"]
+                storeFile = file(filePath)
+                storePassword = keystoreProperties["password"]
+            } else {
+                String storeFilePath = System.getenv("STORE_FILE")
+                println("storeFilePath: " + storeFilePath)
+                if (storeFilePath != null) {
+                    println("using key properties from environment")
+                    keyAlias = System.getenv("KEY_ALIAS")
+                    keyPassword = System.getenv("KEY_PASSWORD")
+                    storeFile = file(storeFilePath)
+                    storePassword = System.getenv("STORE_PASSWORD")
+                } else {
+                    println("No storeFile provided, release builds are using your local debug keystore")
+                }
+            }
+        }
+    }
+
+    buildTypes {
+        debug {
             signingConfig = signingConfigs.debug
+            applicationIdSuffix = ".debug"
+            versionNameSuffix = "-DEBUG"
+        }
+        release {
+            signingConfig = signingConfigs.release
```

Comment:
This proguard file is for sure not needed here?

Reply:
I've re-added it for the default proguard rules to be used on release builds.

Reply:
@dangeross [Create proguard-rules.pro](https://github.com/breez/l-breez/pull/33/commits/adbc7274914eabfcad12fb35f61f5adeebcad103)